ansible/roles/build.rpms.rhel/files/rpm-build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+subscription-manager register --org "${ORG_ID}" --activationkey "${ACT_KEY}"
```

Comment:
```suggestion
subscription-manager register --org="${ORG_ID}" --activationkey="${ACT_KEY}"
```

I believe these options need an `=` symbol. and doesn't subscription-manager beed to run under sudo or directly as root?

Reply:
forget the comment about sudo: the commands are running as root in the container anyways.